packages/avatar/test/visual/lumo/avatar.test.js

```diff
@@ -35,6 +35,30 @@ describe('avatar', () => {
     await visualDiff(div, 'abbr');
   });
 
+  it('name theme-xlarge', async () => {
```

Comment:
Should probably use `abbr` instead of `name` here and below. Also the test above should use `it('abbr')`.